README.md

```diff
@@ -21,46 +36,65 @@ Now with the proof generated by ZooKyc, Gerry goes back to ZooLender (in `gpc-ve
-  - I didn't build a server side app for this, because the verification code would be similar. But in reality, the proof needs to be verified on the server side too.
+#### `pod-issuer-client`
 
-### Turborepo
+- http://localhost:3000, deployed at https://zukyc-issuer-client.vercel.app.
+- This is the frontend of POD issuers (ZooGov, ZooDeel and ZooAdmin).
+- It includes:
+  - `app/gov`: ZooGov
+  - `app/deel`: ZooDeel
+  - `app/debug`: ZooAdmin
+- In real world, those would be three independent websites.
 
-This project was bootstrapped with [Turborepo starter](https://github.com/vercel/turbo/tree/main/examples/basic) by running the following command:
+#### `pod-issuer-server`
 
-```sh
-npx create-turbo@latest -e with-yarn
-```
+- http://localhost:8080, deployed at https://zukyc-issuer-server.vercel.app.
+- This is the server for pod-issuer-client. It provides the server-side APIs to issues PODs to the user.
+- It includes:
+  - `routes/gov.ts`: server-side APIs for ZooGov
+  - `routes/deel.ts`: server-side APIs for ZooDeel
+  - `routes/debug.ts`: server-side APIs for ZooAdmin
+- Similarly, in real world, those would be three independent server-side applications issuing PODs independently.
+- A server is needed because in real world, the server would have a user database; and the server would hold a signing key for issuing PODs.
 
-## Getting started
+### ZooKyc
 
-### Build
+#### `gpc-prover-client`
 
-To build all apps and packages, run the following command:
+- http://localhost:3002, deployed at https://zukyc-gpc-prover-client.vercel.app.
+- This is a frontend app which manages Semaphore identity, PODs, and uses the GPC library to generate ZK proofs based on proof requests.
 
-```
-cd zukyc
-yarn build
-```
+### ZooLender
+
+#### `gpc-verifier`
+
+- http://localhost:3001, deployed at https://zukyc-gpc-verifier.vercel.app.
+- This is a frontend app which provides the proof request, and uses the GPC library to verify ZK proofs.
+- In this demo, the server side code is not included, because the proof verification code would be similar. But in real world, the proof needs to be verified on the server side too.
+
+## Local development
 
-### Develop
+### Running the project
 
-To develop all apps and packages, run the following command:
+In the root of this project, execute the following commands locally.
 
 ```
-cd zukyc
+# installs dependencies for all apps and packages in this repository
+yarn
+
+# this will copy all the environment variable file .env.example into an adjacent file .env or .env.local
+yarn localenv
+
+# this will run a docker image for Vercel KV store (a durable Redis database) local development
+# you need to install docker first and make sure it is running, see this: https://docs.docker.com/engine/install/
+# the Vercel KV store is currently used by `pod-issuer-server` `gov` and `deel` routes, as well as
+# `gpc-verifier` for storing nullifier hash.
+yarn localdb
```

Comment:
hmm, i'm not sure, the pod-arg repo used to have a .env.local file with all those env vars set. I don't want to expose those here because it is a public repo.

Reply:
I was commenting on "yarn localdb" in particular, which I never ran when I set up this repo.

Reply:
It is possible your `KV_REST_API_URL` and `KV_REST_API_TOKEN` in .env.local and .env files point to the Vercel KV store.
Otherwise, if it points to the local redis, you will see errors when issuing PODs in `gov` and `deel` routes; also when you try to verify a proof in `ZooLender`.